warehouse/manage/views/organizations.py

```diff
@@ -190,65 +206,38 @@ def manage_organizations(self):
-    def create_organization(self):
+    @view_config(
+        request_method="POST",
+        request_param=CreateOrganizationApplicationForm.__params__,
+    )
+    def create_organization_application(self):
         # Organizations must be enabled.
         if not self.request.organization_access:
             raise HTTPNotFound()
 
-        form = CreateOrganizationForm(
+        form = CreateOrganizationApplicationForm(
             self.request.POST,
             organization_service=self.organization_service,
+            user=self.request.user,
+            max_apps=self.request.registry.settings[
+                "warehouse.organizations.max_outstanding_applications"
+            ],
         )
 
         if form.validate():
             data = form.data
-            organization = self.organization_service.add_organization(**data)
-            organization.record_event(
-                tag=EventTag.Organization.CatalogEntryAdd,
-                ip_address=self.request.remote_addr,
-                request=self.request,
-                additional={"submitted_by_user_id": str(self.request.user.id)},
-            )
-            organization.record_event(
-                tag=EventTag.Organization.OrganizationCreate,
-                ip_address=self.request.remote_addr,
-                request=self.request,
-                additional={"created_by_user_id": str(self.request.user.id)},
-            )
-            self.organization_service.add_organization_role(
-                organization.id,
-                self.request.user.id,
-                OrganizationRoleType.Owner,
-            )
-            organization.record_event(
-                tag=EventTag.Organization.OrganizationRoleAdd,
-                ip_address=self.request.remote_addr,
-                request=self.request,
-                additional={
-                    "submitted_by_user_id": str(self.request.user.id),
-                    "role_name": "Owner",
-                    "target_user_id": str(self.request.user.id),
-                },
-            )
-            self.request.user.record_event(
-                tag=EventTag.Account.OrganizationRoleAdd,
-                ip_address=self.request.remote_addr,
-                request=self.request,
-                additional={
-                    "submitted_by_user_id": str(self.request.user.id),
-                    "organization_name": organization.name,
-                    "role_name": "Owner",
-                },
+            organization = self.organization_service.add_organization_application(
+                **data, submitted_by=self.request.user
             )
+
             send_new_organization_requested_email(
                 self.request, self.request.user, organization_name=organization.name
             )
             self.request.session.flash(
                 "Request for new organization submitted", queue="success"
```

Comment:
Hmmm, we're kinda split. Requests are how we talk about them on the user side, and application on the backend/admin. Not great, but I think we want to show the user "Request" here since the badge shows "Request submitted"